source/_posts/2017/2017-06-01-thank-you-david.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: 40
-title: "Thank you David"
+title: "Thank you David Grudl"
```

Comment:
I consider my own (similar) post as huge step in my life.  @TomasVotruba thank you for reminder!

Reply:
Anyways, would you mind renaming the file, adding surname `grudl` in it?

Reply:
Thanks for last kick of courage that helped me to do it :)

Any open-soucing plans with that post? :P